src/vs/workbench/contrib/chat/browser/chatEditing/chatEditingTextModelChangeService.ts

```diff
@@ -117,6 +123,17 @@ export class ChatEditingTextModelChangeService extends Disposable {
 		this._register(toDisposable(() => {
 			this.clearCurrentEditLineDecoration();
 		}));
+
+		this._register(autorun(r => this.updateLineChangeCount(this._diffInfo.read(r))));
+	}
+
+	private updateLineChangeCount(diff: IDocumentDiff) {
+		this.lineChangeCount = 0;
+		for (const change of diff.changes) {
```

Comment:
I don't completely understand how `movedLines` in the diff relates to `changes`, but I couldn't manage to get a diff calculation result with `movedLines`, so I just didn't worry about them.